### ISSUE TITLE
Forum monitoring

### ISSUE THREAD

**Issue by Remo** [ortic/concrete5-forum#14]
It would be nice if one could subscribe to a whole forum and receive notifications whenever a new topic (and message?) is posted.

**Comment by mnakalay** [comment]
Would you agree that this would require queuing?

**Comment by Remo** [comment]
This would certainly help, especially if there are a lot of subscribers, but most people don't even use a proper queueing system which concrete5 which is a shame.

**Comment by mnakalay** [comment]
I was thinking about that article you write about using C5 queuing for emails :)
So just a normal queuable job with C5's normal queuing system. Would that work?

**Comment by Remo** [comment]
yes and no, I wouldn't want a job to be mandatory for the forumto work.

I'm a big fan of Laravel regarding mail queues. If you don't run redis, beanstalk it will simply run in sync mode and work just fine, it will only block the user for a bit. If this bothers you, you active a queueing system and everything is going through there.

To me, this should be part of the core, not something we handle in every package. An async mail function.
